config/environments/production.rb

```diff
@@ -31,5 +31,21 @@
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
 
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.perform_deliveries = true
+
+  config.action_mailer.default_url_options =
+    { host: 'ec2-18-218-243-148.us-east-2.compute.amazonaws.com' }
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
+    authentication: :plain,
+    enable_starttls_auto: true,
+    user_name: 'prodobro111@gmail.com',
+    password: 'pro12345678'
+  }
```

Comment:
its bad practice to save email and password here... you have to use global vars for it, and  set them when server starts